src/NumericalAlgorithms/DiscontinuousGalerkin/Actions/ImposeBoundaryConditions.hpp

```diff
@@ -57,7 +74,8 @@ namespace Actions {
 ///      - External<typename system::variables_tag>
 ///
 /// \see ReceiveDataForFluxes
-template <typename Metavariables>
+template <typename Metavariables,
+          bool ApplyUnlessOnlyOutgoingCharSpeeds = false>
```

Comment:
This name has convoluted logical flow that makes it hard to understand the implications of setting this parameter to true or false. I would prefer something more straightforward that avoids the "unless"... perhaps like "SkipBcWhenCharSpeedsAllOutgoing" ?